.npmignore

```diff
@@ -9,5 +9,5 @@
 !README.md
 !binding.gyp
 !index.js
-!index.d.ts
+!*.d.ts
```

Comment:
This change is unnecessary since `ext` is already included.

Reply:
Fixed